app/views/view-solution-page.test.js

```diff
@@ -0,0 +1,176 @@
+import request from 'supertest';
+import express from 'express';
+import cheerio from 'cheerio';
+import { App } from '../../app';
+
+const createDummyApp = (context) => {
+  const app = new App().createApp();
+
+  const router = express.Router();
+  const dummyRouter = router.get('/', (req, res) => {
+    res.render('view-solution-page.njk', context);
+  });
+
+  app.use(dummyRouter);
+
+  return app;
+};
+
+
+describe('view solution', () => {
+  it('should render back-link component', (done) => {
+    const context = {};
+    const app = createDummyApp(context);
+    request(app)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+        const backLink = $('[data-test-id="view-solution-page-back-link"]');
+        expect(backLink.length).toEqual(1);
+        done();
+      });
+  });
+
+  it('should render the foundation tag if isFoundation is true', (done) => {
+    const context = {
+      isFoundation: true,
+    };
+    const app = createDummyApp(context);
+    request(app)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+        const orgName = $('[data-test-id="solution-foundation-tag"]');
```

Comment:
change the name of the variable from orgName

Reply:
why?

Reply:
oh right haha